src/bin/clean.rs

```diff
@@ -0,0 +1,23 @@
+use std::io;
+use std::io::Write;
+use std::fs;
+use clap::{App, ArgMatches, SubCommand};
+use mdbook::MDBook;
+use mdbook::errors::Result;
+use get_book_dir;
+
+// Create clap subcommand arguments
+pub fn make_subcommand<'a, 'b>() -> App<'a, 'b> {
+    SubCommand::with_name("clean").about("Delete built book")
```

Comment:
Should this accept a `--dest-dir` argument? (like `mdbook build` and friends)

Reply:
Fixed.

Reply:
WIP: did.